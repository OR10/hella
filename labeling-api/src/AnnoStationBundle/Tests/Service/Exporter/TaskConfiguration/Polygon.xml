<?xml version="1.0" encoding="UTF-8" ?>
<labelTaskConfig shape="polygon" minimalVisibleShapeOverflow="16" minimalHeight="0">
    <class id="occlusion" name="Occlusion">
        <value id="occlusion-0" name="0%"/>
        <value id="occlusion-1" name="&lt; 25%"/>
        <value id="occlusion-2" name="25% - 50%"/>
        <value id="occlusion-3" name="&gt; 50%"/>
    </class>
    <class id="truncation" name="Truncation">
        <value id="truncation-0" name="0%"/>
        <value id="truncation-1" name="&lt; 25%"/>
        <value id="truncation-2" name="25% - 50%"/>
        <value id="truncation-3" name="&gt; 50%"/>
    </class>
    <class id="direction" name="direction">
        <value id="direction-back" name="Back"/>
        <value id="direction-back-right" name="Back Right"/>
        <value id="direction-back-left" name="Back Left"/>
        <value id="direction-right" name="Right"/>
        <value id="direction-front" name="Front"/>
        <value id="direction-front-right" name="Front Right"/>
        <value id="direction-front-left" name="Front Left"/>
        <value id="direction-left" name="Left"/>
    </class>
</labelTaskConfig>