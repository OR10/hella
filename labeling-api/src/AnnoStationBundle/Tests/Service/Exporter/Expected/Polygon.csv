"UUID","FrameNumber","Polygon","occlusion","truncation","direction"
"0735702d52300b44aea6a35ad00564c4","23","148,144;248,244;348,344","occlusion-0","truncation-0","direction-back"
"0735702d52300b44aea6a35ad00564c4","45","448,444;548,544;648,644","occlusion-0","truncation-0","direction-back"